### ISSUE TITLE
Consider packaging just JRE

### ISSUE THREAD

**Issue by sideeffffect** [chainguard-images/jdk#5]
Consider please also having a flavor/variant of this image which contains only the JRE (runtime environment), not the whole big JDK.
The image will be much smaller.

E.g. https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.4.1%2B1/OpenJDK17U-jre_x64_alpine-linux_hotspot_17.0.4.1_1.tar.gz

**Comment by amouat** [comment]
Yes, that's absolutely our plan. We'll have the JDK image for compiling and JRE for running images.

**Comment by tuananh** [comment]
waiting for jdk/jre package from wolfi https://github.com/wolfi-dev/os/issues/51

**Comment by Footur** [comment]
It would be nice to have headless package version of the JRE like [openjdk17-jre-headless](https://pkgs.alpinelinux.org/package/v3.16/community/x86_64/openjdk17-jre-headless).
